src/command_modules/azure-cli-network/azure/cli/command_modules/network/zone_file/make_zone_file.py

```diff
@@ -54,32 +46,58 @@ def make_zone_file(json_zone_file, template=None):
         "uri":     [ uri records ]
     }
     """
+    import sys
+    import azure.cli.command_modules.network.zone_file.record_processors as record_processors
+    from six import StringIO
+
+    old_stdout = sys.stdout
```

Comment:
Good point. It was mostly because I was lazy and didn't want to add the file param to each call to print. But based on your feedback, I will go ahead and do that so it doesn't cause unforeseen problems. Thanks :)